thesaurus/thesaurus/TrainViewModel.cs

```diff
@@ -43,16 +43,16 @@ public TrainViewModel(TrainModel model)
         #region Handlers
 
         /// <summary>
-        /// 
+        /// Handler for Window Loaded event. Stores reference to Window on VM.
         /// </summary>
-        /// <param name="obj"></param>
-        private void OnWindowLoaded(Window obj)
+        /// <param name="win">Train Window.</param>
+        private void OnWindowLoaded(Window win)
```

Comment:
I think I missed this when you added it, what's the use case in your mind for this?